site/zenodo_rdm/migrator/dump_communities.py

```diff
@@ -0,0 +1,11 @@
+import yaml
+from pathlib import Path
+from invenio_communities.communities.records.models import CommunityMetadata
+community_map = {comm.slug: str(comm.id) for comm in CommunityMetadata.query.all()}
+streams_path = str(Path('site/zenodo_rdm/migrator/streams.yaml').absolute())
+streams = {}
+with open(streams_path, 'r') as fp:
```

Comment:
I guess there is no way to overwrite only one key, that's why read-then-write?

Reply:
didn't find one, but still this is a temp solution for now....